octoprint_mrbeam/templates/ready_to_laser.jinja2

```diff
@@ -1,60 +1,73 @@
-<div id="ready_to_laser_dialog" class="modal hide fade" data-backdrop="static" data-test="mrbeam-ui-modal-ready-to-laser">
-
-    <div data-bind='visible: !is_cooling_mode() || !state.interlocksClosed()'>
+<div id="ready_to_laser_dialog" class="modal hide fade" data-backdrop="static"
+     data-test="mrbeam-ui-modal-ready-to-laser">
         <div class="modal-header">
-            <a href="#" class="close" data-bind='visible: is_rtl_mode(), click: cancel_btn' aria-hidden="true">&times;</a>
-            <h3 data-bind='visible: is_rtl_mode()'>{{ _('Ready To Laser?') }}</h3>
-            <h3 data-bind='visible: !is_rtl_mode()'>{{ _('Pause') }}</h3>
-        </div>
-        <div class="modal-body">
-            <img data-bind='visible: !is_fan_connected()' src="/plugin/mrbeam/static/img/airfilter.jpg">
-            <img data-bind='visible: !state.interlocksClosed() && is_fan_connected()' src="/plugin/mrbeam/static/img/closelid.jpg">
-            <img data-bind='visible: state.interlocksClosed() && is_fan_connected()' src="/plugin/mrbeam/static/img/onebutton.jpg">
-            <h5 data-bind='visible: !is_fan_connected()'>{{ _('Can not detect air filter system. Is it plugged in?') }}</h5>
-            <h5 data-bind='visible: !state.interlocksClosed() && is_fan_connected()'>{{ _('Please close the lid completely before we can proceed.') }}</h5>
-            <h5 data-bind='visible: state.interlocksClosed() && is_fan_connected()'>{{ _('Everything is ready to go.') }}</h5>
-            <h5 data-bind='visible: is_fan_connected() && is_rtl_mode() && jobTimeEstimationCalculated()'>{{ _('Estimated duration') }}:
-                <span data-bind="text: jobTimeEstimationString()" style="font-size:1.2em"></span>
-            </h5>
+            <a href="#" class="close" data-bind='visible: !is_pause_mode(), click: cancel_btn'
+               aria-hidden="true">&times;</a>
+            <h3 data-bind='visible: !is_pause_mode()'>{{ _('Ready To Laser?') }}</h3>
+            <h3 data-bind='visible: paused'>{{ _('Pause') }}</h3>
+            <h3 data-bind='visible: cooling'>{{ _('Cooling') }}</h3>
```

Comment:
is_pause_mode is with parenthesis while paused and cooling are not. Why the inconsistency here especially that both are computed observables

Reply:
same in other parts of this file